internal/playable/playable.go

```diff
@@ -0,0 +1,3 @@
+package playable
```

Comment:
Is `doc` a better package name?

Reply:
Agree. Which one do you think is better, `doc` or `godoc`?

Reply:
It's a valid concern.

Reply:
How about moving the package to `registry/remote/internal/doc` so that the `doc` package is specific.

Reply:
`registry/example_test.go` is also using this `ExampleUnplayable` value. If we move the package to `registry/remote/internal/doc`, will raise `invalid use of internal package` error in `registry/example_test.go`. How about moving to `registry/internal/doc`?

Reply:
> How about moving to `registry/internal/doc`?

That makes sense.

Reply:
Rebased and made requested change.